docs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "docs",
-  "version": "54.12.0",
```

Comment:
this was done via `git revert ff49172c0534bebcd9941929ff70eb3d340efe80`

Reply:
Just googled `git revert` vs `git reset`, and confirmed this is right commend we should use here. Good to learn more git knowledge from reviewing this PR!

Reply:
One more question I have: How would this `revert` enable us to re-run the release script in circle? I don't quite understand the connection between here and circle.